requirements.txt

```diff
@@ -1,4 +1,5 @@
 PyYAML==5.4.1
+http://archive.ubuntu.com/ubuntu/pool/main/p/python-apt/python-apt_2.1.3ubuntu1.tar.xz; sys_platform == 'linux'
```

Comment:
As this really needs to come from the host, I'd just leave it at `python-apt` for now.

Reply:
It seems that only a very old version (0.7.8) is available at PyPI, unsure if 2.1.3 available anywhere else.

Reply:
Ah yes, i mean that it needs to be `apt install`d.  It must match the host apt version.

Reply:
this is not as easy from a python "library" point of view

Reply:
I'd say we should look into hosting our own wheel